package.json

```diff
@@ -5,19 +5,23 @@
     "email": "roger@bufferoverflow.ch"
   },
   "scripts": {
-    "build": "babel src/ --out-dir lib/ --copy-files --ignore ___tests___ --source-maps",
-    "build:docker": "docker build -t verdaccio-gitlab . --no-cache",
-    "flow": "flow",
+    "flow": "flow check",
     "commitmsg": "commitlint -e $GIT_PARAMS",
     "license": "license-checker --onlyAllow 'Apache-2.0; Apache License, Version 2.0; BSD; BSD-2-Clause; BSD-3-Clause; ISC; MIT; Unlicense; WTFPL; CC-BY-3.0; CC0-1.0' --production",
-    "lint": "flow && eslint . && eclint check $(git ls-files) && markdownlint README.md",
-    "prepublish": "in-publish && yarn lint && yarn build || not-in-publish",
+    "lint": "yarn flow && eslint . && eclint check $(git ls-files) && markdownlint README.md",
+    "prepublish": "in-publish && yarn lint && yarn code:build || not-in-publish",
     "release:major": "changelog -M && git commit -a -m 'docs: updated CHANGELOG.md' && yarn version major && git push origin && git push origin --tags",
     "release:minor": "changelog -m && git commit -a -m 'docs: updated CHANGELOG.md' && yarn version minor && git push origin && git push origin --tags",
     "release:patch": "changelog -p && git commit -a -m 'docs: updated CHANGELOG.md' && yarn version patch && git push origin && git push origin --tags",
-    "start": "yarn build && NODE_PATH=$NODE_PATH:.. node lib/verdaccio.js"
+    "start": "yarn code:build && cross-env NODE_PATH=$NODE_PATH:.. BABEL_ENV=registry babel-node build/verdaccio.js",
+    "code:build": "cross-env BABEL_ENV=registry babel src/ --out-dir build/ --copy-files --ignore ___tests___ --source-maps",
+    "code:docker-build": "cross-env BABEL_ENV=registry-docker babel src/ --out-dir build/ --copy-files --ignore ___tests___ --source-maps",
+    "build:docker": "docker build -t verdaccio-gitlab . --no-cache",
```

Comment:
Any particular reason to use `--no-cache` flag? `--no-cache` can slow down the build significantly

Reply:
It's basically safeguarding: it ensures consistent builds everywhere. In my experience the docker cache is a source of problems, specially in development, since the `RUN` directives need to change to actually trigger a rebuild of the layer, and you sometimes need it :-)

I actually do myself the manual `docker build ...` command lots of times to make it faster. If you see the need, we can add an additional command without it. But in practice the main use is the CI, since now @bufferoverflow has enabled auto-building in the docker hub.

It's also used in verdaccio itself (before my refactor).